### PR TITLE
collect client logs with proper namespace

### PR DESCRIPTION
**Pull request by yati1998**
after storageclient being clusterscoped
the logs for client cluser were not getting collected. This commit adds the changes to include all the logs and other resources

### CODE REVIEW

**Comment by yati1998** [comment]
```
yatipadia:quay-io-ypadia-odf-must-gather-sha256-d2f0285d745ab6f9e0a68fa8e9e27f2cc9c276e561022c6f56253867ac62f80a$ ls
cluster-scoped-resources  event-filter.html  namespaces  storageclaim    timestamp
csidriver                 gather-debug.log   noobaa      storageclients  validatingwebhookconfiguration

yatipadia:quay-io-ypadia-odf-must-gather-sha256-d2f0285d745ab6f9e0a68fa8e9e27f2cc9c276e561022c6f56253867ac62f80a$ cd namespaces/
yatipadia:namespaces$ ls
all  openshift-operators  openshift-storage-client
```

File: collection-scripts/gather_odf_client [hunk 1]
```diff
@@ -4,8 +4,7 @@
 # If it is not defined, use PWD instead
 BASE_COLLECTION_PATH=${BASE_COLLECTION_PATH:-"$(pwd)"}
 
-STORAGE_CLIENT_NAMESPACE=$(oc get storageclients -A --no-headers | awk '{print $1}')
-CLIENT_OPERATOR_NAMESPACE=$(oc get subscriptions -A --no-headers | grep ocs-client-operator | awk '{print $1}')
+STORAGE_CLIENT_NAMESPACE=$(oc get deploy -l app=ocs-client-operator -A --no-headers | awk '{print $1}')
```
Comment (leelavg) [inline collection-scripts/gather_odf_client:7]: pls use the var name as `CLIENT_OPERATOR_NAMESPACE`

**Review by leelavg — COMMENTED** [review]
LGTM

**Review by black-dragon74 — APPROVED** [review]
LGTM

**Pull request merged**